src/Cli/dotnet/commands/dotnet-workload/install/IInstaller.cs

```diff
@@ -33,6 +33,19 @@ internal interface IInstaller : IWorkloadManifestInstaller
         void ReplaceWorkloadResolver(IWorkloadResolver workloadResolver);
 
         void Shutdown();
+
+        /// <summary>
+        /// Delete the install state file at the specified path.
+        /// </summary>
+        /// <param name="path">The full path of the default.json install state file.</param>
+        void DeleteInstallState(string path);
```

Comment:
Consider changing these methods to take an SdkFeatureBand instead of the path.  Then the message sent to the server can include the feature band instead of the path, and the server can just validate that the SdkFeatureBand is valid and then construct the path using `WorkloadInstallType.GetInstallStateFolder`.

Reply:
Pushed new changes to address this. We already have the ability to pass the feature band around when sending messages to the server. I think I got everything removed that we no longer need.

Once this is merged, we should close the 8.0.1xx PR and then backport this instead if we decide to take it for 8.0.1xx